applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -523,10 +442,9 @@ void SmallStrainUPwDiffOrderElement::CalculateDampingMatrix(MatrixType&        r
 
     const SizeType ElementSize = NumUNodes * Dim + NumPNodes;
 
-    // Compute Mass Matrix
-    MatrixType MassMatrix(ElementSize, ElementSize);
-
-    this->CalculateMassMatrix(MassMatrix, rCurrentProcessInfo);
+    MatrixType MassMatrix = GeoTransportEquationUtilities::CalculateMassMatrixDiffOrder(
```

Comment:
Since this is a local variable, this should be snake case (problem of course is that it seems this class very inconsistent with that right now, maybe we can fix it only in the functions we touch?)
```suggestion
    MatrixType mass_matrix = GeoTransportEquationUtilities::CalculateMassMatrixDiffOrder(
```

Reply:
Thank you. Yesterday I made the same comments to Mohamed ;(